pyproject.toml

```diff
@@ -6,9 +125,89 @@ requires = ["setuptools",
             "extension-helpers"]
 build-backend = 'setuptools.build_meta'
 
+[tool.setuptools]
+include-package-data = true
+
+[tool.setuptools.packages.find]
+include = ["astropy*"]
+exclude = ["astropy._dev*"]
+namespaces = false
+
+[tool.setuptools.package-data]
+"*" = [
+    "data/*",
+    "data/*/*",
+    "data/*/*/*",
+    "data/*/*/*/*",
+    "data/*/*/*/*/*",
+    "data/*/*/*/*/*/*",
```

Comment:
I know this was a direct transfer to pyproject, but perhaps this can be cleaned up?

Reply:
Also fine to push to a future PR

Reply:
I agree! However, this is attempting do as much as possible a 1-to-1 port to the `pyproject.toml` file.

So I think this can wait until a future PR.